packages/api-client/index.ts

```diff
@@ -32,7 +32,7 @@ class APIClient {
     return responseClass ? plainToInstance(responseClass, res) : res;
   }
 
-  constructor(baseURL = "/api") {
+  constructor(baseURL = "http://localhost:3001/api") {
```

Comment:
This would be changed back for prod as both the backend and frontend would be running on the same URL, but for testing purposes the backend and frontend are on different ports.

Reply:
No longer needed as you can now use http://localhost:3002